src/main/java/edu/iu/terracotta/service/caliper/impl/CaliperServiceImpl.java

```diff
@@ -117,6 +117,21 @@ public void sendAssignmentStarted(Submission submission, SecuredInfo securedInfo
         org.imsglobal.caliper.entities.resource.Assessment assessment = prepareAssessment(submission, securedInfo);
         Attempt attempt = prepareAttempt(submission, actor, assessment);
         String uuid = "urn:uuid:" + UUID.randomUUID();
+
+        Map<String, Object> extenstions = new HashMap<>();
+
+        Long assignmentId = submission.getAssessment().getTreatment().getAssignment().getAssignmentId();
+        Long assessmentId = submission.getAssessment().getAssessmentId();
+        Long treatmentId = submission.getAssessment().getTreatment().getTreatmentId();
+
+        extenstions.put("terracotta_assignment_id",assignmentId);
+        extenstions.put("terracotta_submission_id",submission.getSubmissionId());
+        extenstions.put("terracotta_submission_id",submission.getSubmissionId());
```

Comment:
Looking at TCOTA-443, I don't see a requirement to add submission id.But condition id and exposure id are listed in the issue but missing here.